packages/grpc-js/src/index.ts

```diff
@@ -127,6 +127,7 @@ export {
   LogVerbosity as logVerbosity,
   Status as status,
   ConnectivityState as connectivityState,
+  Propagate as propagate,
```

Comment:
These should also be exported from the main package so they can be accessed off the `grpc` namespace like other packages (like the native node one).

Reply:
This is the main package.

Reply:
Ah, my bad. the way GitHub folded the code, it looked like it was part of a separate import statement.